### PR TITLE
Update mfa_setup.md

### PR DESCRIPTION
**Pull request by markmrets**
adding photos once I get the ability to

### CODE REVIEW

**Review by markmrets — COMMENTED** [review]
@BryanGower This can be reviewed

File: mfa_setup.md
Comment (BryanGower) [inline mfa_setup.md:1]: I would reword this a little: 

M-RETS has updated our multi-factor authentication (MFA) sign-in options for enhanced security. As part of this update, the use of the Authy app has been discounted. Instead, users with MFA enabled will need to request their authentication code via SMS or Voice call. This MFA method ensures secure access by combining your credentials with a time-based one-time password obtained on your trusted mobile device.

**Review by BryanGower — COMMENTED** [review]
@markmrets I added some comments with suggested edits
